### PR TITLE
Add notes from 102nd meeting, 2017-08-03; Update CWP status.

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Update with the notes of the last meeting. Conditions CWP ready for review.

### CODE REVIEW

File: organization/_posts/2017-08-03-startup.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  - What is the best way to do this? Semi-regular general HSF meetings should be revived in September (Wednesday's 17h seems ok or use the Startup Thursday slot).
+- The SFT group has been invited to present the result of the CWP and next steps to the Scientific Computing Forum (SCF) established earlier this year by Eckhard Elsen. The meeting will take place on October 27th.
+- New job at CERN to increase the effort in ROOT development. [Details](https://jobs.web.cern.ch/job/12661).
+
+## CWP
+
+-   #### Simulation
+
+    - Executive summary was sent to the experiments yesterday (computing coordinators and simulation contacts). In good shape from the group. 2 weeks for extended people to contribute. Final convergence soon after 16 August.
+    - Should there be a glossary? Maybe an editorial team decision. Should be looked by document readers - please not any areas where jargon needs defined.
```
Comment (hegner) [inline organization/_posts/2017-08-03-startup.md]: not -> note

File: organization/_posts/2017-08-03-startup.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+-   #### MachineLearning
+
+    -   From Sergei: "We are in final stages of review editing, filling in missing pieces like references etc. There are a couple missing pieces which we hope to get completed by the end of the week.”
+
+-   #### Software Development
+
+    -   Document very close to conclusion, but still lacking a little in Agile section and case studies. Some feedback from reviewers to work in (thanks!). It’s proving hard to make progress over the summer months (holidays then consequent work backlogs…). We should close by *force majeure* 17 August.
```
Comment (hegner) [inline organization/_posts/2017-08-03-startup.md]: work *on*

File: organization/_posts/2017-08-03-startup.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+-   #### Next Steps
+
+    - Pete Elmer was pulling out names - try to identify people even if it’s a few weeks before work will begin.
+
+## Activity updates
+
+### Training
+
+- WikiToLearn is getting input from the GirdKA school people, ([website](https://en.wikitolearn.org/GridKa)).
```
Comment (hegner) [inline organization/_posts/2017-08-03-startup.md]: GirdKA -> GridKA

File: organization/_posts/2017-08-03-startup.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+-   #### Next Steps
+
+    - Pete Elmer was pulling out names - try to identify people even if it’s a few weeks before work will begin.
+
+## Activity updates
+
+### Training
+
+- WikiToLearn is getting input from the GirdKA school people, ([website](https://en.wikitolearn.org/GridKa)). 
+- John H: New Springer journal has some interesting papers in the pipeline. If you know a interesting topics we can invite contributions (ideas: Fred James - PRNGs; Rene Braun - Data Analysis; SWAN and notebooks; Machine learning). Editors would like to have early contributions that describe concepts.
```
Comment (hegner) [inline organization/_posts/2017-08-03-startup.md]: a*n* interesting

Comment (hegner) [inline organization/_posts/2017-08-03-startup.md]: "Braun" -> "Brun"

File: organization/_posts/2017-08-03-startup.md (hunk 1 above)
Comment (graeme-a-stewart) [inline organization/_posts/2017-08-03-startup.md]: Done

**Pull request merged**